### DISCUSSION TITLE
Do we have state wise timeout for the task in airflow v2.2.2 ?

### DISCUSSION THREAD

**Discussion by vivek-zeta** [apache/airflow#21925] [Q&A]
I'm looking for feature to mark task as fail if it stays in particular state for some amount of time.
e.g. if task stays in queue state for 5 min I want to terminate the task.

Please help me if anything like this is currently supported in airflow v2.2.2

**Comment by potiuk** [comment]
I think there is no such feature. Feel free to open a feature request.
